core/src/components/button/button.ios.scss

```diff
@@ -149,19 +149,26 @@
 }
 
 ::slotted(ion-icon[slot="icon-only"]) {
-  font-size: 1.125em;
+  /**
+    * The values were provided through a native iOS app.
+    * min font size: 13.5px, default font size: 18px, max font size: 40.5px
+    */
+  font-size: dynamic-font-clamp(0.75, 18px, 2.25, 1em);
 }
 
 :host(.button-small.button-has-icon-only) {
   // TODO(FW-6053): replace em value with the min-height variable.
   min-width: clamp(23px, 2.16em, 54px);
   // TODO(FW-6053): replace em value with the min-height variable.
   min-height: clamp(23px, 2.16em, 54px);
-  
 }
 
 :host(.button-small) ::slotted(ion-icon[slot="icon-only"]) {
-  font-size: 1.4em;
+  /**
+    * The values were provided through a native iOS app.
+    * min font size: 13px, default font size: 17px, max font size: 38px
+    */
+  font-size: dynamic-font-clamp(0.63, 20.93px, 1.84, 1em);
```

Comment:
Why is the default font size 20.93 when it says the default font size is 17 in the comments?

Reply:
This is due to `ion-button`, the parent, having a different font size than the typical 16px.